test/run_test.py

```diff
@@ -69,6 +67,9 @@
 # skip < 3.6 b/c fstrings added in 3.6
 if PY36:
     TESTS.append('jit_py3')
+    TESTS.append('dist_autograd_spawn')
```

Comment:
Can we add comments here about why we need these tests to be only >=3.6 temporarily?